src/components/components/Component.js

```diff
@@ -61,7 +61,7 @@ export default class Component extends React.Component {
     const componentData = this.props.component;
 
     if (isSingleProperty(componentData.schema)) {
-      const componentName = componentName.toLowerCase();
+      const componentName = this.props.name.toLowerCase();
```

Comment:
I thought that I needed to ensure that `componentName` is lowercase to access the schema for each component and the user could do `GEOMETRY`and it should still works, isn't it?